Zend/tests/readonly_function.phpt

```diff
@@ -7,10 +7,34 @@ function readonly() {
     echo "Hi!\n";
 }
 
+class A {
+    const readonly = 'Const hi!';
+
+    static function readonly() {
+        echo "Static hi!\n";
+    }
+}
+
+class B {
+    public $readonly = 'Prop hi!';
+
+    function readonly() {
+        echo "Instance hi!\n";
+    }
+}
+
+$b = new B();
+
 readonly();
-readonly ();
```

Comment:
Does using ``readonly ()`` with the whitespace now break? Or is it unnecessary due to the move to the parser?

Reply:
It's just unnecessary since this is no longer special-cased. This should still work. I can keep it if you prefer.

Reply:
No harm keeping it then :)